app/views/idv/by_mail/letter_enqueued/show.html.erb

```diff
@@ -4,47 +4,40 @@
         current_step: :get_a_letter,
         locale_scope: 'idv',
         class: 'margin-x-neg-2 margin-top-neg-4 tablet:margin-x-neg-6 tablet:margin-top-neg-4',
-      ) %>
+      )
+  %>
 <% end %>
 
 <% self.title = t('titles.idv.come_back_soon') %>
 
 <%= image_tag(
       asset_url('come-back.svg'),
-      size: '140', class: 'display-block margin-x-auto margin-bottom-2',
+      size: '140', class: 'display-block margin-left-0 margin-bottom-2',
       alt: t('idv.images.come_back_later')
-    ) %>
+    )
+%>
 
-<%= render PageHeadingComponent.new(class: 'text-center').with_content(t('idv.titles.come_back_later')) %>
+<%= render PageHeadingComponent.new(class: 'text-left').with_content(t('idv.titles.come_back_later')) %>
```

Comment:
Left alignment should be the default IIRC, so we could remove the class altogether?

```suggestion
<%= render PageHeadingComponent.new.with_content(t('idv.titles.come_back_later')) %>
```

Reply:
Removed. Thanks.